src/main/kotlin/blackjack/model/ResultEvaluator.kt

```diff
@@ -0,0 +1,66 @@
+package blackjack.model
+
+data class DealerResult(var wins: Int = 0, var losses: Int = 0, var draws: Int = 0) {
+    override fun toString(): String {
+        return "Dealer: $wins wins, $losses losses, $draws draws"
+    }
+}
+
+data class PlayerResult(var win: Boolean = false, var draw: Boolean = false, val name: String = "") {
+    override fun toString(): String {
+        if (win) return "$name: win"
+        if (draw) return "$name: draw"
+        return "$name: loss"
```

Comment:
It seems like the model depends on the view.
To follow the Single Responsibility Principle, consider moving this logic into the view.